docs/getting_started/installation/installation.rst

```diff
@@ -39,17 +39,19 @@ in order to get all of the dependencies::
 .. note::
     The above installation is for OSX and Unix. If you are using Windows, use environment-win.yml instead of environment-dev.yml
```

Comment:
I think this would be a bit overkill, as it is already highlighted in blue (see image below).    Therefore, I suggest leaving it as is.  

![note_cutout](https://user-images.githubusercontent.com/65550266/125363989-a4aa4880-e33f-11eb-885b-cb120e906e3d.png)

Reply:
oh I see, I viewed those file as mardown file, so they were not rendered as pretty